com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -297,34 +197,33 @@ private void FixedUpdate()
                 return;
             }
 
-            if (CanUpdateTransform())
+            if (IsNetworkStateDirty)
             {
-                m_NetworkPosition.Value = m_CurrentPosition;
-                m_NetworkRotation.Value = m_CurrentRotation;
-                m_NetworkWorldScale.Value = m_CurrentScale;
+                if (CanUpdateTransform)
+                {
+                    UpdateNetworkState();
+                }
+                else
+                {
+                    ApplyNetworkState(m_NetworkState.Value);
+                }
             }
-            else if (m_CurrentPosition != m_OldPosition ||
-                m_CurrentRotation != m_OldRotation ||
-                m_CurrentScale != m_OldScale
-            )
-            {
-                Debug.LogError($"Trying to update transform's position for object {gameObject.name} with ID {NetworkObjectId} when you're not allowed, please validate your {nameof(NetworkTransform)}'s authority settings. It's also possible you have other systems updating your transform's position, such as a rigid body for example.", gameObject);
```

Comment:
I'd really keep that this error message. It's important to guide users so they don't have silent errors.

Reply:
there is no error condition in the new flow. we simply do not care. it's not an error?

Reply:
from what I tested with your branch, the transform just doesn't move if it doesn't have the authority. It looks very frustrating. "Why does my code updating that object's transform doesn't do anything??"

Reply:
huh, let me try a couple of things there.